packages/react-instantsearch-hooks-web/src/widgets/Pagination.tsx

```diff
@@ -28,6 +28,18 @@ export function Pagination({
   showLast,
   padding,
   totalPages,
+  translations = {
+    first: '‹‹',
+    previous: '‹',
+    next: '›',
+    last: '››',
+    page: (currentPage: number) => String(currentPage),
+    ariaFirst: 'First',
+    ariaPrevious: 'Previous',
+    ariaNext: 'Next',
+    ariaLast: 'Last',
+    ariaPage: (currentPage: number) => `Page ${currentPage}`,
```

Comment:
I'd say it depends on why builders use `translations`, if it's primarily for translating into another language it will just make code more complicated with having to add a switch or a map or whatever conditional